bot_file.py

```diff
@@ -182,9 +191,9 @@ def stop_listening(self, channel: str, pattern: str) -> None:
         self.notify.forget_pattern(person, pattern)
         self.notify.save_to_file(self.notify_file)
 
-    def time_to_show(self, channel: str, person: str) -> None:
-        """ notify a person about a message """
-        if channel.startswith('D'):
+    def ping_person(self, channel: str, person: str) -> None:
+        """ notify a person about a message. Ignore direct messages """
```

Comment:
Will come back to this in another PR - you're right, it should probably be scoped to the right channel